theano/sandbox/cuda/corr_gemm.cu

```diff
@@ -286,7 +307,8 @@ CudaNdarray* corrMM(CudaNdarray *const bottom,
         for (int n = 0; n < batchSize; n++) {
             // First, im2col
             im2col(bottom->devdata + n * bottom_stride, nChannels, bottomHeight,
-                    bottomWidth, kH, kW, padH, padW, dH, dW, col->devdata);
+		   bottomWidth, kH, kW, dilH, dilW,
+		   padH, padW, dH, dW, col->devdata);
```

Comment:
nitpick: I'd prefer either the previous indentation (8 spaces = a double indent) or one space more than you have now (a visual indent). Indenting by 6 spaces is not common.